src/Slides.js

```diff
@@ -0,0 +1,31 @@
+import { URL } from 'url';
+
+const getTrimmedPathName = pathname => pathname.replace(/\/+$/, '');
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  if (!['slides.com', 'www.slides.com', 'team.slides.com'].includes(host)) {
```

Comment:
This can be simplified to

```ts
return ['slides.com', 'www.slides.com', 'team.slides.com'].includes(host) &&
  getTrimmedPathName(pathname).split('/').filter(Boolean).length === 2
```

Since
```ts
if(!condition1) {
  return false;
}

return condition2;
```
is the same as

```ts
if(condition){
  return condition2;
} else {
  return false;
}
```
which is the same as
```ts
if(condition1 && condition2) {
  return true;
} else {
  return false;
}
```
so we could take `condition1` & `condition2` together like

```ts
return condition1 && condition2;
```